frontend/components/vault/VaultLiquidationTransactionTable.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+    <div class="flex flex-col space-y-1 text-gray-700 dark:text-gray-100">
+        <div class="flex justify-between">
+            <div>Proximity to liquidation threshold</div>
+            <div>
+                <span v-if="vaultTransaction.proximityToLiquidation < 0" class="text-green-500 mr-1">
+                    (Ready for liquidation)
+                </span>
+                <span v-if="vaultTransaction.proximityToLiquidation">
```

Comment:
In which case there can be no `proximityToLiquidation`? Please remove this `v-if` together with empty `v-else`

Reply:
When the vault was liquidated there is no more `proximityToLiquidation`. At least when the type is created it does not exist anymore. 

Aside form that I find having fallbacks better then the UI throwing an error if a user ever runs into an edge case.

Reply:
If the vault was liquidated we do not have these values anymore. There could be an edge case where this happens so I added it as an extra safety measure.